controllers/metrics.go

```diff
@@ -0,0 +1,58 @@
+package controllers
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"sigs.k8s.io/controller-runtime/pkg/metrics"
+)
+
+// prometheus metrics gauges
+var (
+	APIServerReadyMetric = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "data_science_pipelines_application_apiserver_ready",
```

Comment:
@gmfrasca can you add a section to the readme and list out these metrics that are expoed at the metrics endpoint?